src/Microsoft.Azure.WebJobs.Host/Loggers/TraceEventExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    internal static class TraceEventExtensions
+    {
+        private const string FunctionDescriptorKey = "MS_FunctionDescriptor";
+        private const string FunctionInvocationIdKey = "MS_FunctionInvocationId";
+        private const string HostIdKey = "MS_HostId";
+
+        public static void AddFunctionInstanceDetails(this TraceEvent traceEvent, Guid hostId, FunctionDescriptor descriptor, Guid functionId)
+        {
+            traceEvent.Properties[HostIdKey] = hostId;
```

Comment:
Wondering what the scenario is for HostId? This will be the same for all logs, right? So I'm wondering how this would be useful for someones custom trace writer. If they wanted HostId they could access that themselves by passing it into their custom tracer on construction.